program-workflows/adjust-recency-data-sync-frequency.md

```diff
@@ -0,0 +1,21 @@
+##HOW FREQUENTLY TO SEND RECENCY DATA
```

Comment:
@ODORA0 do not use capitals in a heading

Reply:
@ssmusoke done